app/models/item.rb

```diff
@@ -0,0 +1,7 @@
+class Item < ApplicationRecord
+  belongs_to :poll
+  has_many :votes, dependent: :delete_all
+
+  after_create_commit -> { broadcast_append_to self.poll.party, locals: { item: self }, target: "poll_#{self.poll.id}_items_container" }
```

Comment:
I think we don't need `locals: { item: self }` since we are getting the items from `poll.items.each do |item|` I tested it removing this line and it worked